README.md

```diff
@@ -1,49 +1,91 @@
 
-This project's origin is [here](https://github.com/weaviate/weaviate-examples/tree/main/face-recognition-app)
+This project's origin is [here](https://github.com/weaviate/weaviate-examples/tree/main/face-recognition-app)
 
-## Description
+## [Description](https://github.com/weaviate-tutorials/DEMO-face-recognition#description)
 
-This example application spins up a Weaviate instance using the
-[img2vec-neural](https://github.com/semi-technologies/i2v-pytorch-models)
-module, imports a few sample images (you can add your own images, too!) and
-provides a very simple search frontend in [React](https://reactjs.org/) using
-the [Weaviate JS Client
-](https://www.semi.technology/developers/weaviate/current/client-libraries/javascript.html)
+This example application spins up a Weaviate instance using the [img2vec-neural](https://github.com/semi-technologies/i2v-pytorch-models) module, imports a few sample images (you can add your own images, too!) and provides a very simple search frontend in [React](https://reactjs.org/) using the [Weaviate JS Client](https://www.semi.technology/developers/weaviate/current/client-libraries/javascript.html)
 
 (TO DO: Add demo video)
 
-### Used technology stack
-This demo uses the [RESNET50](https://pytorch.org/hub/nvidia_deeplearningexamples_resnet50/) model from [pytorch.org](https://pytorch.org).
+### [Used technology stack](https://github.com/weaviate-tutorials/DEMO-face-recognition#used-technology-stack)
 
+This demo uses the [RESNET50](https://pytorch.org/hub/nvidia_deeplearningexamples_resnet50/) model from [pytorch.org](https://pytorch.org/).
+
+## Need for this Application
+Facial Recognition has been an amazing catalyst for advancements in the field of Object Detection and Computer Vision. It has been seamlessly weaved into our everyday life from using FaceID to unlock your phone, to automated album segregation based on individuals in a picture. It is even used in Institutions for marking attendance. 
+
+## Deep Dive into our Solution
+Previously, it used to cost a lot of resources to run Facial Recognition Models and perform Image Searches. There were no Vector Databases, remember :)
+
+Now, Vector Databases, like Weaviate do almost all the heavy lifting. Weaviate is so powerful that it can perform an Image Search in an instant. 
+
+So, what is happening under the hood?
+
+Three main steps:
+1. Weaviate Database Setup
+2. Image Vectorization
+3. Querying our Database
+
+#### 1. Weaviate Database Setup
+The test data, we have for populating the database contains 5 generated images of people. Each person will have three properties: `filename`, `image`, and `name`. These properties are hence defined in the schema, which is the structure for storing data in Weaviate. Schema is initialized by either running `create_schema.py` or `create_schema.sh`.  
+
+Here’s the schema for this application:
+![create_schema](./images/create_schema.jpg)
```

Comment:
I think it would be nicer, if this was a code block instead of an image of a code block. Also, if you could add a reference to the file the schema is located as you did with the TS code snippet below, that would be awesome!